synapse/rest/client/account.py

```diff
@@ -882,8 +882,7 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
 
         response = {
             "user_id": requester.user.to_string(),
-            # MSC: https://github.com/matrix-org/matrix-doc/pull/3069
-            "org.matrix.msc3069.is_guest": bool(requester.is_guest),
```

Comment:
I think we've been erring on having some overlap with the stable and unstable version, even if just for a release.

Reply:
👍 helpfully though, doing this at least identified that the tests also need fixing.